cl_manycore/executables/bsg_manycore_spmd_loader.cpp

```diff
@@ -0,0 +1,75 @@
+#include "bsg_manycore_run_spmd_program.hpp"
+#include <bsg_manycore.h>
+#include <bsg_manycore_printing.h>
+#include <stdlib.h>
+#include <stdio.h>
```

Comment:
Would it be useful to separate cpp headers and c headers with ifdef?

Reply:
Quick learner :p

Reply:
I hate to be spicy about this but... the only difference it makes (that I know of) is that the C library functions are imported as part of the `std` namespace -- which we really don't use when calling C library functions because... why would we? It's borderline sociopathic to make your own function literally called `printf` or `exit`. Worse still, this only applies to standard C library functions -- not to `unistd.h` or `endian.h` or `elf.h` or a whole bunch of other system headers we might want.

So sure, I guess I can change it; but I think we have much better things to worry about than whether or not we can call `printf` like `std::printf`.

Reply:
ha :D